data/mods/StatsThroughSkills/modinfo.json

```
@@ -25,7 +25,7 @@
   {
     "type": "skill_boost",
     "stat": "int",
-    "skills": [ "barter", "computer", "cooking", "electronics", "fabrication", "firstaid", "speech" ],
+    "skills": [ "computer", "cooking", "electronics", "fabrication", "firstaid", "speech", "chemistry", "spellcraft" ],
```

`spellcraft` is not part of vanilla and `StatsThroughSkills ` is independent from `Magiclysm`

```suggestion
    "skills": [ "computer", "cooking", "electronics", "fabrication", "firstaid", "speech", "chemistry" ],
```

---

Barter was not a valid skill name for some time and didn't result in any problems for the time it was in the list, keeping "spellcraft" on it would allow compatibility between both mods without any side-effect for people that don't use magiclysm.

---

> 
> 
> Barter was not a valid skill name for some time and didn't result in any problems for the time it was in the list, keeping "spellcraft" on it would allow compatibility between both mods without any side-effect for people that don't use magiclysm.

It is an oversight that code does not validate skill strings.

---

> `spellcraft` is not part of vanilla and `StatsThroughSkills ` is independent from `Magiclysm`
> 

Yes, but spellcraft is a skill. Every other skill boosts starts, but spellcraft doesn't. That's inconsistent.